packaging/rhel/syslog-ng.spec

```diff
@@ -577,6 +577,9 @@ fi
 %{_datadir}/%{name}/tools/
 
 %changelog
+* Fri Aug  15 2019 Laszlo Budai <laszlo.budai@balabit.com> - 3.23.1-1
```

Comment:
There is an unnecessary second whitespace between `Aug` and `15`.
Aug 15 should be `Thu`.